summernote/summernote
# Issue 951 Hide popover menu on blur

#### What's this PR do?

- hides the popover menu when the summernote control is blurred

#### How should this be manually tested?

- create a page with a summernote control in air mode
- click on some text to open the popover menu
- click away from the summernote control

The popover menu should be hidden

#### What are the relevant tickets?

This fixes issue https://github.com/summernote/summernote/issues/951
